CHANGELOG.md

```diff
@@ -148,7 +232,7 @@ It is just modified to be a minimally buildable state, it is for testing purpose
 
 ### Core gems
 
-* async-http v0.60.2 (update)
+* async-http v0.61.0 (update)
```

Comment:
Oops, you changed the previous release 5.0.1.

Reply:
Ohhh... Sorry for being so careless...
Fixed!